cmd/calyptia/create_core_instance_gcp.go

```diff
@@ -1,19 +1,115 @@
 package main
 
 import (
-	"errors"
-
+	"context"
+	"fmt"
+	"github.com/calyptia/cli/gcp"
+	"github.com/calyptia/core-images-index/go-index"
 	"github.com/spf13/cobra"
+	"log"
 )
 
-func newCmdCreateCoreInstanceOnGCP(config *config) *cobra.Command {
+func newCmdCreateCoreInstanceOnGCP(config *config, client gcp.Client) *cobra.Command {
+	var (
+		coreInstanceName      string
+		zone                  string
+		projectID             string
+		machineType           string
+		environment           string
+		tags                  []string
+		noHealthCheckPipeline bool
+		coreInstanceVersion   string
+		externalIP            string
+		sshKeyPath            string
+		sshUser               string
+	)
 	cmd := &cobra.Command{
 		Use:     "gcp",
 		Aliases: []string{"google", "gce"},
-		Short:   "Setup a new core instance on Google Compute Engine (TODO)",
+		Short:   "Setup a new core instance on Google Compute Engine",
+		Long:    "Setup a new core instance on Google Compute Engine, you need to be authenticated with gcloud cli \n",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return errors.New("not implemented")
+			ctx := context.Background()
```

Comment:
Recently, I noticed you can get the context from `cmd`.
```suggestion
			ctx := cmd.Context()
```